engine/include/cubos/engine/gl/pps/bloom.hpp

```diff
@@ -0,0 +1,82 @@
+#ifndef CUBOS_ENGINE_GL_PPS_BLOOM_HPP
+#define CUBOS_ENGINE_GL_PPS_BLOOM_HPP
+
+#include <cubos/core/gl/render_device.hpp>
+
+#include <cubos/engine/gl/pps/pass.hpp>
+
+#include <glm/glm.hpp>
+
```

Comment:
You can remove the whitespace between the glm include and the std include, we usually split it into engine, core and external includes.

Reply:
Fixed